orchagent/orchdaemon.cpp

```diff
@@ -39,6 +39,10 @@ NatOrch *gNatOrch;
 
 bool gIsNatSupported = false;
 
+#define SWITCH_SHUTDOWN_LOG_INTERVAL_IN_SECS 60
```

Comment:
Suggest to place this `#define` near the others (lines 17-18)

Reply:
done